tests/tests/custom_authority_tests.cpp

```diff
@@ -1257,6 +1259,571 @@ BOOST_AUTO_TEST_CASE(custom_auths) { try {
+         //              [
+         //                {
+         //                  "member_index": 1,
+         //                  "restriction_type": 0,
+         //                  "argument": [
+         //                    8,
+         //                    "1.3.5"
+         //                  ],
+         //                  "extensions": []
+         //                }
+         //              ]
+         //            ],
+         //            "extensions": []
+         //          }
+         //        ],
+         //        [
+         //          {
+         //            "member_index": 3,
+         //            "restriction_type": 10,
+         //            "argument": [
+         //              39,
+         //              [
+         //                {
+         //                  "member_index": 1,
+         //                  "restriction_type": 0,
+         //                  "argument": [
+         //                    8,
+         //                    "1.3.6"
+         //                  ],
+         //                  "extensions": []
+         //                }
+         //              ]
+         //            ],
+         //            "extensions": []
+         //          }
+         //        ]
+         //      ]
+         //    ],
+         //    "extensions": []
+         //  }
+         //]
+
+         // Broadcast the authorization
+         trx.clear();
+         trx.operations = {authorize_limit_orders};
+         sign(trx, alice_private_key);
+         PUSH_TX(db, trx);
+
+
+         // Custom Authority 2: Sell BCOINs to buy ACOIN
+         vector<restriction> sell_b1_rx = {restriction(amount_to_sell_index, FUNC(attr),
+                                                       vector<restriction>{restriction(asset_id_index, FUNC(eq),
+                                                                                       asset_id_type(
+                                                                                               bcoin1.id.instance()))})};
+         vector<restriction> sell_b2_rx = {restriction(amount_to_sell_index, FUNC(attr),
+                                                       vector<restriction>{restriction(asset_id_index, FUNC(eq),
+                                                                                       asset_id_type(
+                                                                                               bcoin2.id.instance()))})};
+         vector<restriction> sell_b3_rx = {restriction(amount_to_sell_index, FUNC(attr),
+                                                       vector<restriction>{restriction(asset_id_index, FUNC(eq),
+                                                                                       asset_id_type(
+                                                                                               bcoin2.id.instance()))})};
+         restriction sell_bcoin_rx = restriction(dummy_index, FUNC(logical_or),
```

Comment:
Maybe replace one  of the `logical_or` with a list function?

Reply:
This was a very good suggestion Peter.  By using the `in` function, the full set of pairs could be combined and only consume 9 restrictions with almost limitless assets on either side of the permitted exchange sets.

Another benefit is that 9 restrictions is less than the likely 10 restriction limit per custom authority which means that the restricted trading can be described by one custom authority rather than 2 custom authorities.